### PR TITLE
Multiple improvements, including multi-authors at home

### PR DESCRIPTION
**Pull request by lorinkoz**
Collaboration is important, as well as giving credit to all authors!

### CODE REVIEW

File: _includes/multi_author_info.html [hunk 1]
```diff
@@ -1,20 +1,31 @@
-
 <div class="author-container">
 {% for writer in page.author %}
     {% assign author = site.data.authors[writer] %}
     <img src="{{ site.url }}/assets/images/{{ author.image|default: 'placeholder_author.jpg' }}" alt="" class="author-image big" />
     <div class="author-info">
-        <div class="author-name">{{ author.name }}</div>  
+        <div class="author-name">{{ author.name }}</div>
+        {% if forloop.first %}
+        <div class="article-date">{{ page.date | date: "%B %e, %Y" }}</div>
```
Comment (lorinkoz) [inline _includes/multi_author_info.html:8]: Date was floating around. Now is located behind the first author.

File: index.html [hunk 2]
```diff
@@ -11,39 +11,54 @@
...
+                          <div class="article-date">{{ post.date | date: "%B %e, %Y" }}</div>
+                        </div>
+                      {% else %}
+                        {% assign writer = site.data.authors[post.author] %}
+                        <img src="{{ site.url }}/assets/images/{{ writer.image|default: 'placeholder_author.jpg' }}" alt="{{ writer.name }}" class="author-image">
+                        <div class="author-info">
+                          <div class="author-name">{{ writer.name }}</div>
+                          <div class="article-date">{{ post.date | date: "%B %e, %Y" }}</div>
+                        </div>
+                      {% endif %}
```
Comment (lorinkoz) [inline index.html:54]: Support for multiple authors in the frontpage cards.

File: index.html [hunk 3]
```diff
@@ -11,39 +11,54 @@
...
+                        {% comment %} author is not a string, but an array {% endcomment %}
+                        {% for author in post.author %}
+                          {% assign writer = site.data.authors[author] %}
+                          <img src="{{ site.url }}/assets/images/{{ writer.image|default: 'placeholder_author.jpg' }}" alt="{{ writer.name }}"
+                            class="author-image">
+                        {% endfor %}
+                        <div class="author-info">
+                          <div class="author-name">
+                            {%- for author in post.author -%}
+                              {%- if forloop.last -%}&nbsp;&amp;&nbsp;{%- elsif forloop.first -%}{%- else -%},&nbsp;{%- endif -%}
```
Comment (lorinkoz) [inline index.html:41]: Author names are joined using commas and ampersands:
- John
- John & Jane
- John, Paul & Jane

**Review by glsignal — APPROVED** [review]
Solid improvements! Thanks Lorenzo!

**Pull request merged**